docs/data/base/components/switch/switch.md

```diff
@@ -0,0 +1,48 @@
+---
+product: base
+title: React Switch component
+components: SwitchUnstyled
+githubLabel: 'component: switch'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#switch
+packageName: '@mui/base'
+---
+
+# Switch
+
+<p class="description">Switches toggle the state of a single setting on or off.</p>
+
+Switches are the preferred way to adjust settings on mobile.
+The option that the switch controls, as well as the state it's in, should be made clear from the corresponding inline label.
```

Comment:
This is part is sounding a bit confusing to me. 🤔

Reply:
@danilo-leal I (hopefully) made it clearer. Does it look better for you now?

Reply:
Oh yeah, much better! Thanks for tweaking this one up :)